mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodAppDelegate.m

```diff
@@ -192,31 +207,46 @@ - (void)startSentry {
     options.dsn = @"https://960f8b8e574c46e3be385d60ce8e1fea@o1005580.ingest.sentry.io/5983522";
     options.releaseName = releaseName;
     options.environment = keymanVersionInfo.sentryEnvironment;
-    //options.debug = YES;
   }];
 }
 
-- (void)setPostLaunchKeymanSentryTags {
-  NSString *keyboardFileName = [self.kmx.filePath lastPathComponent];
-   os_log_info([KMLogs keyboardLog], "initial kmx set to %{public}@", keyboardFileName);
+/**
+ * Determine whether to force sentry events.
+ * When true, then typing certain characters will cause Sentry events or crashes to occur.
+ *
+ * To enable, set the flag in UserDefaults
+ * `defaults write keyman.inputmethod.Keyman KMForceSentryError 1`
+ * Also, use the SIL Euro Latin keyboard and type in the Stickies app
+ */
+- (BOOL)canForceSentryEvents {
+  NSString* const testKeyboardName = @"sil_euro_latin.kmx";
+  NSString* const testClientApplicationId = @"com.apple.Stickies";
+  BOOL canForce = NO;
```

Comment:
Can you include this change in the PR description? (Makes the developer testing more visible).

Ideally, this could have been separated from the Sentry breadcrumbs PR so it shows in HISTORY.md.

Reply:
will do